plenum/server/config_req_handler.py

```diff
@@ -113,25 +113,30 @@ def get_taa_digest(self, version: Optional[str] = None,
         if res is not None:
             return res.decode()
 
+    # TODO return object as result instead
     def get_taa_data(self, digest: Optional[str] = None,
                      version: Optional[str] = None,
-                     isCommitted: bool = True) -> Optional[Dict]:
+                     isCommitted: bool = True) -> Tuple[Optional[Dict], Optional[str]]:
+        data = None
         if digest is None:
```

Comment:
If digest is not None, then `None` data will be returned. Is it intended?

Reply:
right, fixed